proposal.md

```diff
@@ -2487,6 +2655,38 @@ template <> struct hash<json_v1::json>
 *Effect:* The template specialization to calculate the hash value for the specified
 JSON value.
 
+<a name="limitations"></a>
+## Limitations and Interoperability

```

Comment:
Regarding Unicode: yes, UTF-8 is sufficient.

The rest: I don't know. I would leave the items as a reminder that if we make a limitation, we should acknowledge it.